src/main/java/com/etsy/statsd/profiler/Profiler.java

```diff
@@ -16,19 +18,44 @@
 
     private final Reporter<?> reporter;
     // CONTAINER_ID=container_e273_1510780065801_7737_01_000042
-    private String[] tags = {};
+    private String[] tags = null;
 
     private long recordedStats = 0;
     public Profiler(Reporter reporter, Arguments arguments) {
         Preconditions.checkNotNull(reporter);
         this.reporter = reporter;
         handleArguments(arguments);
 
+        Collection<String> tagsList = new ArrayList<>();
         String containerId = System.getenv("CONTAINER_ID");
         if (containerId != null) {
             String applicationId = getApplicationId(containerId);
-            tags = new String[]{"container_id:" + containerId, "application_id:" + applicationId};
+            tagsList.add("container_id:" + containerId);
+            tagsList.add("application_id:" + applicationId);
         }
+
+        String luigiTaskFamily = arguments.remainingArgs.get("LUIGI_TASK_FAMILY");
+        if (luigiTaskFamily != null) {
+            tagsList.add("luigiTaskFamily:" + luigiTaskFamily);
```

Comment:
What does LuigiTaskFamily mean here?

Reply:
Good question:
`TaskFamily` is the LuigiTask (aka job) that has been executed eg: `BuildPCCModel` so the we can have a select-box is DD to select the jobs that we want to have report on